package.json

```diff
@@ -60,11 +60,12 @@
     "conventional-changelog": "^1.1.0",
     "dgeni": "^0.4.7",
     "dgeni-packages": "^0.16.5",
+    "firebase": "^3.7.2",
     "firebase-admin": "^4.1.2",
     "firebase-tools": "^2.2.1",
-    "firebase": "^3.7.2",
     "fs-extra": "^2.0.0",
     "glob": "^7.1.1",
+    "google-closure-compiler": "^20170218.0.0",
```

Comment:
I haven't experienced issues with this such that I have an opinion one way or the other. Is it reasonable to leave it until we run into a problem?